src/algokit/cli/tui/init/screens/example_selector_screen.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+
+from textual import on
+from textual.app import ComposeResult
+from textual.containers import Vertical
+from textual.screen import Screen
+from textual.widgets import Footer, Header, Label, ListItem, ListView
+
+from algokit.core.init import _load_algokit_examples
+
+if TYPE_CHECKING:
+    from algokit.cli.tui.init.example_selector import ExampleSelector
+
+
+class ChooseExampleScreen(Screen):
+    app: "ExampleSelector"
+    """A screen that displays available example templates based on chosen framework."""
+
+    CSS = """
+    .margin-bottom-1 {
+        margin-bottom: 1;
+    }
+    .examples-screen {
+        height: auto;
+        overflow-y: auto;
+        width: 100%;
+    }
+    """
+    SUB_TITLE = "Add Example"
```

Comment:
```suggestion
    SUB_TITLE = "Init"
```
Nitpick: I wonder if "Init" is better to match the command?

Reply:
Yeah i think so. I changed it to `Initialize Example`